content/blog/2017-04-06-interview-with-europace.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Interview with Europace
+created_at: 2017-04-06
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Tobias Gesellchen from
+Europage talks about how they discovered Prometheus.*
+
+
+## Can you tell us about Europace does?
+
+[Europace AG](https://www.europace.de/) develops and operates the web-based
+EUROPACE financial marketplace, which is Germany’s largest platform for
+mortgages, building finance products and personal loans. A fully integrated
+system links about 400 partners – banks, insurers and financial product
+distributors. Several thousand users execute some 35,000 transactions worth a
+total of up to €4 billion on EUROPACE every month.  Our engineers regularly
+blog at [http://tech.europace.de/](http://tech.europace.de/) and
+[@EuropaceTech](@EuropaceTech).
+
+## What was your pre-Prometheus monitoring experience?
+
+[Nagios](https://www.nagios.org/)/[Icinga](https://www.icinga.com/) are still
+in use for other projects, but with the growing number of services and higher
+demand for flexibility we looked for other solutions. Due to Nagios and Icinga
+being more centrally maintained, Prometheus matched our aim to have the full
+DevOps stack in our team and move specific responsibilities from our
+infrastructure team to the project members.
+
+## Why did you decide to look at Prometheus?
+
+Through our activities in the [Docker Berlin
+community](https://www.meetup.com/Docker-Berlin/) we had been in contact with
+[SoundCloud] (https://soundcloud.com/) and [Julius
+Volz](https://twitter.com/juliusvolz), who gave us a good overview. The
+combination of flexible Docker containers with the highly flexible labels based
+concept convinced us give Prometheus a try.  The Prometheus setup was easy
+enough, the Alertmanager worked for our needs, so that we didn’t see any reason
```

Comment:
run-on sentence / missing conjunction:

s/the Alertmanager/and the Alertmanager/